packages/pagination/addon/styles/components/_nucleus-pagination.scss

```diff
@@ -0,0 +1,66 @@
+.nucleus-pagination {
+  padding: 12px 8px;
+
+  .nucleus-paginator {
+    background: $color-milk;
+
+    ul {
+      list-style: none;
+      margin: 0 !important; // to prevent other vendor CSS from overriding ul in general
+      padding: 6px 0 !important;
+      line-height: 1;
+    }
+
+    li {
+      position: relative;
+    }
+
+    &__number {
+      display: inline-block;
+      margin: 0 !important;
```

Comment:
Why `!important`?

Reply:
Some old CSS from the yore (I have no idea who wrote it) 😅, will remove.